### ISSUE TITLE
/sbr autotab only shows create|loadbrush tab completes, none of the sub commands.

### ISSUE THREAD

**Issue by TomLewis** [eldoriarpg/SchematicBrushReborn#268]
Im trying to figure out how to remove a brush from an item in-game, apparently it binds to all of that item, and not directly to that one specific one via data! Im looking at /sbr in game to try and figure out commands, if I miss-spell a command, it gives me a list of whats part of the plugin, but tab only shows create & loadbrush as tab options.

![2023-03-28_16 17 06](https://user-images.githubusercontent.com/144046/228286088-678f4154-c2dd-4427-abcd-74c626fd9b02.png)

**Comment by TomLewis** [comment]
So its just /unbind, vanilla WE/FAWE. 

Maybe this should be part of the text UI

**Comment by rainbowdashlabs** [comment]
SBR is just a normal world edit brush in the end. You can unbind it like every other brush you use with world edit. Maybe I should educate the users about it more. Linking a specific brush to an item instead of an item type is not in the scope of world edit I think. Ideally it would be added there.

Adding unbind to the ui might have unexpected side effects. Especially since the UI only displays a copy of the brush. The unbind command would then probably just unbind the brush from the item in your hand, which might not be the brush anymore you expect to unbind.

The sbr command only shows commands which can be executed directly. All other shown subcommands are bound to a button inside the text ui.

**Comment by TomLewis** [comment]
Ahh right that makes sense, internal commands for other parts of the plugin.

Well, adding unbind to the wiki would help, I spent 20mins trying to figure it out! 

The "manage your presets" command is wrong on this page too, I think the R and P are the wrong way around?
https://github.com/eldoriarpg/SchematicBrushReborn/wiki/Commands

**Comment by rainbowdashlabs** [comment]
Yeah that might be :D need to update this in general I think. There are a few new commands which aren't listed there yet.
